src/mixins/breadcrumb-mixin.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { CurrentAccountIF } from '@/interfaces'
+import { Routes } from '@/enums'
+import { useAuthenticationStore, useBusinessStore, useConfigurationStore, useRootStore } from '@/stores'
+
+/**
+ * Mixin that provides some useful Name Request utilities.
+ */
+@Component({})
+export default class BreadcrumbMixin extends Vue {
```

Comment:
I changed this to a mixin so that it could directly access the store getters.

I know it's sometimes possible to access the store externally, ie:
```
const authenticationStore = useAuthenticationStore()
const currentAccound = authenticationStore.getCurrentAccount
```

However, I found that I was not seeing the latest store value that I cared about (isNoRedirect) -- it was set to True but I was getting False. So, I'm using a mixing and the Getter mechanism.

Reply:
PS This mixin is used exclusively by App.vue so using a mixin isn't polluting `this` any more than necessary.